Tests/helper.py

```diff
@@ -45,7 +45,7 @@ def upload(a, b):
 
 else:
     try:
-        import test_image_results
+        import TestImageResults
```

Comment:
I believe this is importing https://pypi.org/project/test-image-results/, https://github.com/wiredfool/test-image-results.

If anything, it might make sense to just remove it.

Reply:
Good point, I've reverted it.